src/sentry/api/endpoints/group_autofix_setup_check.py

```diff
@@ -17,12 +17,15 @@
 from sentry.models.group import Group
 from sentry.models.organization import Organization
 from sentry.models.project import Project
+from sentry.models.promptsactivity import PromptsActivity
 from sentry.seer.signed_seer_api import sign_with_seer_secret
 
 logger = logging.getLogger(__name__)
 
 from rest_framework.request import Request
 
+feature_name = "seer_autofix_setup_acknowledged"
```

Comment:
if this enablement applies to all future Seer features too, should we remove "autofix" from the name?